Author: <kees@ubuntu.com>
Description: Fix FTBFS with openssl.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/ssh.c
+++ b/ssh.c
@@ -16,6 +16,7 @@
 #include <openssl/ssl.h>
 #include <openssl/err.h>
 #include <openssl/rand.h>
+#include <openssl/md5.h>
 
 #include <err.h>
 #include <errno.h>
